anisoap/representations/ellipsoidal_density_projection.py

```diff
@@ -646,3 +653,46 @@ def transform(self, frames, show_progress=False, normalize=True):
             return normalized_features
         else:
             return features
+
+    def power_spectrum(self, ell_frames, AniSOAP_HYPERS=None):
+        mycg = ClebschGordanReal(AniSOAP_HYPERS["max_angular"])
```

Comment:
It might be better to allow this as an argument to the function, as precomputing the CG can cut down on compute time.